src/ResourceManager/Resources/Commands.ResourceManager/Cmdlets/Components/ApiVersionHelper.cs

```diff
@@ -108,7 +109,7 @@ private static string[] GetApiVersionsForResourceType(AzureContext context, stri
                 ? "/providers"
                 : string.Format("/subscriptions/{0}/providers", defaultSubscription.Id);
 
-            var providers = PaginatedResponseHelper.Enumerate(
+            ResourceProviderDefinition[] providers = PaginatedResponseHelper.Enumerate(
```

Comment:
Why? I prefer var here, instead of the type. That's the convention we follow in the project